docs/pre.sh

```diff
@@ -1,4 +1,29 @@
 #!/bin/bash
 
 cp -a ../rpc/openapi/ .vuepress/public/rpc/
-git clone https://github.com/tendermint/spec.git specRepo && cp -r specRepo/spec . && rm -rf specRepo
+
+git clone https://github.com/tendermint/spec.git specRepo
+cp -r specRepo/spec .
+rm -rf specRepo
+
+mkdir 0.32
+git clone -b v0.32.0  --depth 1  https://github.com/tendermint/tendermint.git 32Repo
+cp -r 32Repo/docs 0.32
+rm -rf 32Repo
+rm -rf 0.32/docs/DOCS_README.md
+rm -rf 0.32/docs/README.md
+cd 0.32/docs
+mv * ../
+rm -rf 0.32/docs
+
+cd ../..
+
+mkdir 0.33
+git clone -b v0.33.0  --depth 1  https://github.com/tendermint/tendermint.git 33Repo
+cp -r 33Repo/docs 0.33
+rm -rf 33Repo
+rm -rf 0.33/docs/DOCS_README.md
+rm -rf 0.33/docs/README.md
+cd 0.33/docs
+mv * ../
+rm -rf 0.33/docs
```

Comment:
How does this differ from:

https://github.com/tendermint/tendermint/blob/297573269a81d639675454a5ecbe576b9efbe421/docs/versions#L1-L4

& 

https://github.com/tendermint/tendermint/blob/0aecda68fc35347a3435d00d14f4e3d17cde08d7/Makefile#L190-L198

it seems previous versions are being built twice now?

Reply:
These are two different approaches. Original one checked out different versions and built them as separate websites in separate directories. The new one apparently copies md files into the existing website. Gotta choose one or the other, not both.

Reply:
The current approach doesn't seem to be working on the production. Oh, I'm merely providing a fix for the versions under the Resources.

![image](https://user-images.githubusercontent.com/1021101/94495985-07af6200-01a8-11eb-9216-f43f47f18f9f.png)

Reply:
yea, I never understood why it didn't work. Could you fix the build-docs command to not build and clone multiple versions

Reply:
I'll go ahead and downgrade the theme versions of `v0.32` and `v0.33` branch wrt https://github.com/tendermint/tendermint/pull/5409#issuecomment-700936877

Reply:
if we merge this PR then that PR is not needed right? 

What else is needed to push this PR across the finish line? Should we remove the version file?

Reply:
I'd like to give the makefile approach another try. It didn't work before because there are JS build errors in the versioned branch (v0.33).

So it (almost) silently fails to build the website.

If after fixing the JS errors doesn't work, I'll focus on the latter approach (this PR) which is cloning the versions.